app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -71,7 +71,7 @@ def authenticate(login = account, password = account_password)
       return
     end
 
-    attrs.each { |k, v| attrs[k] = v.force_encoding('UTF-8') }
+    attrs.each { |k, v| attrs[k] = v.force_encoding('UTF-8') unless v == ''}
```

Comment:
Would this be cleaner?
```suggestion
    attrs.each { |k, v| attrs[k] = v.force_encoding('UTF-8') unless v.empty?}
```

Reply:
Maybe, but I wanted to limit this to strings specifically.

Reply:
Is `force_encoding` defined on anything but strings?

Reply:
Probably not.

Reply:
To be more specific: I didn't dig deeper, but I'm quite sure these empty strings are casted `nil` values from somewhere else and in the future that whole LDAP stack needs a makeover in a different place to be more UTF-8 compliant anyway.